scripts/spack/packages/conduit/package.py

```diff
@@ -364,8 +364,18 @@ def hostconfig(self, spec, prefix):
         if cxxflags:
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS", cxxflags))
         fflags = ' '.join(spec.compiler_flags['fflags'])
-        if self.spec.satisfies('%cce'):
+
+        # AXOM EDIT START
+        # Hack to fix -lmpi error for TOSS4 installed examples
+        cfg.write(cmake_cache_entry("conduit_blt_mpi_deps", "mpi"))
+        cfg.write(cmake_cache_entry("CONDUIT_USE_CMAKE_MPI_TARGETS", "OFF"))
+
+        if self.spec.satisfies('%cce') or (f_compiler is not None and ("crayftn" in f_compiler)):
             fflags += " -ef"
+            # Disable executables that fail during link with TOSS4 MPI
+            cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "OFF"))
```

Comment:
@cyrush do you know about this?  This is fine in the short term but a longer term solution would be preferable. Also a variant so we can turn these off would be helpful as well.   We generally turn off all extra things in our packages like examples/tests/etc because we don't use them ourselves and it speed sup the builds.